nav2_rviz_plugins/src/goal_tool.cpp

```diff
@@ -47,11 +47,13 @@ void
 GoalTool::onPoseSet(double x, double y, double theta)
 {
   std::string fixed_frame = context_->getFixedFrame().toStdString();
-
-  navigation_dialog_->startNavigation(x, y, theta, fixed_frame);
-  navigation_dialog_->show();
-  navigation_dialog_->raise();
-  navigation_dialog_->activateWindow();
+  bool start_navigation_result = true;
+  start_navigation_result = navigation_dialog_->startNavigation(x, y, theta, fixed_frame);
+  if (start_navigation_result != false) {
```

Comment:
I agree with Steve's comment: should be

```C++
if (startNavigation(x, y, theta, fixed_frame) {
}

```

The conversation is marked as resolved, but the code doesn't reflect the update.

Reply:
I did the changes in local, I haven't pushed it to the remote yet. Sorry for the confusion. I will keep the conversations open until the pr is merged.